procrastinate/contrib/django/admin.py

```diff
@@ -106,12 +119,19 @@ def short_task_name(self, instance: models.ProcrastinateJob) -> str:
 
     @admin.display(description="Args")
     def pretty_args(self, instance: models.ProcrastinateJob) -> str:
-        indent = 2 if len(instance.args) > 1 or len(str(instance.args)) > 30 else None
-        pretty_json = json.dumps(instance.args, indent=indent)
-        if len(pretty_json) > 2000:
-            pretty_json = pretty_json[:2000] + "..."
+        arg_rows = [
+            format_html(
+                "<tr><td>{key}</td><td>{value}</td></tr>",
```

Comment:
If we go this route, we should at least wrap keys and values in `<code>`, but it that much better than json ?

json is easily copyable.
yaml could be a good middle ground, maybe ? It's readable (but it needs one more dep :( )